web/client-api/src/main/java/io/deephaven/web/client/ide/LspTranslate.java

```diff
@@ -82,4 +79,56 @@ private static Position toJs(
         item.character = (int) src.getCharacter();
         return item;
     }
+
+    private static MarkupContent toJs(
+            final io.deephaven.javascript.proto.dhinternal.io.deephaven.proto.console_pb.MarkupContent src) {
+        final MarkupContent content = new MarkupContent();
+
+        content.kind = src.getKind();
+        content.value = src.getValue();
+        return content;
+    }
+
+    public static SignatureInformation toJs(
+            io.deephaven.javascript.proto.dhinternal.io.deephaven.proto.console_pb.SignatureInformation src) {
+        final SignatureInformation item = new SignatureInformation();
+        item.label = src.getLabel();
+        if (src.hasDocumentation()) {
+            item.documentation = toJs(src.getDocumentation());
+        }
+        if (src.hasActiveParameter()) {
+            item.activeParameter = src.getActiveParameter();
+        }
+
+        final JsArray<ParameterInformation> params = new JsArray<>();
+        final JsArray<io.deephaven.javascript.proto.dhinternal.io.deephaven.proto.console_pb.ParameterInformation> paramsList =
+                src.getParametersList();
+        for (int i = 0; i < paramsList.getLength(); i++) {
+            params.push(toJs(paramsList.getAt(i)));
+        }
+        item.setParameters(params);
+        return item;
+    }
+
+    private static ParameterInformation toJs(
+            final io.deephaven.javascript.proto.dhinternal.io.deephaven.proto.console_pb.ParameterInformation src) {
```

Comment:
in your place, i would probably make these constructors for the various js types, but this isnt a requirement/expectation